cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/Measure.java

```diff
@@ -300,6 +305,14 @@ public SelectedReport hasEvaluatedResourceCount(int count) {
             return this;
         }
 
+        public SelectedReport evaluatedResourceHasNoDuplicateReferences() {
```

Comment:
Do the methods that return SelectedReport need to be public?  There's a warning about a package-protected class being exposed outside its scope.  Consider removing the public modifier.

Reply:
I believe they are needed to be public as they are chained tests called externally

Reply:
Then please make the SelectedReport and other inner classes public since otherwise the external callers won't be able to access those classes.